typescript/infra/package.json

```diff
@@ -67,5 +68,8 @@
     "node": "hardhat node",
     "prettier": "prettier --write *.ts ./src ./config ./scripts ./test",
     "test": "mocha --config ../sdk/.mocharc.json test/agents.test.ts"
+  },
+  "peerDependencies": {
```

Comment:
@yorhodes why did you add this? I don't think it will have any affect given that infra is a private (not published) package

Reply:
it resolved some warnings yarn was giving